app/controllers/api/promos_controller.rb

```diff
@@ -4,21 +4,12 @@ class Api::PromosController < Api::BaseController
 
   api_versions :v1
 
-  def resource
-    super.tap do |af|
-      if story && af.try(:new_record?)
-        af.audio_version_id ||= story.promos.id
-        af.account_id ||= story.account_id
```

Comment:
So these are coming from the request params, provided by using a nested route to get to this call, right?